package.json

```diff
@@ -56,25 +27,56 @@
       "last 1 safari version"
     ]
   },
+  "dependencies": {
+    "axios": "~0.21.4",
+    "bootstrap": "~4.6.1",
+    "react": "~17.0.2",
+    "react-bootstrap": "~2.1.2",
+    "react-dom": "~17.0.2",
+    "react-dropzone": "~11.7.1",
+    "react-password-strength-bar": "~0.4.0",
+    "react-redux": "~7.2.6",
+    "react-router-dom": "~6.2.1",
+    "react-scripts": "~5.0.0",
+    "redux": "~4.1.2",
+    "redux-types": "~2.0.3",
+    "sass": "~1.45.2",
+    "super-tiny-icons": "~0.4.0"
+  },
   "devDependencies": {
-    "@babel/core": "^7.13.14",
-    "@storybook/addon-actions": "6.4.9",
-    "@storybook/addon-essentials": "6.4.9",
-    "@storybook/addon-links": "6.4.9",
-    "@storybook/addon-storyshots": "6.4.9",
-    "@storybook/node-logger": "6.4.9",
-    "@storybook/preset-create-react-app": "3.2.0",
-    "@storybook/react": "6.4.9",
-    "axios-mock-adapter": "1.20.0",
-    "babel-loader": "8.1.0",
-    "cypress": "9.1.1",
-    "eslint-plugin-cypress": "2.12.1",
-    "husky": "7.0.4",
-    "npm-check-updates": "12.0.5",
-    "prettier": "2.5.1",
-    "rc-util": "5.16.1",
-    "react-is": "17.0.2",
-    "react-test-renderer": "17.0.2",
-    "sonarqube-scanner": "2.8.1"
+    "@babel/core": "~7.17.2",
+    "@storybook/addon-actions": "~6.4.19",
+    "@storybook/addon-essentials": "~6.4.19",
+    "@storybook/addon-links": "~6.4.19",
+    "@storybook/addon-storyshots": "~6.4.19",
+    "@storybook/node-logger": "~6.4.19",
+    "@storybook/preset-create-react-app": "~4.0.1",
+    "@storybook/react": "~6.4.19",
+    "@testing-library/jest-dom": "~5.16.2",
+    "@testing-library/react": "~12.1.2",
+    "@testing-library/user-event": "~13.5.0",
+    "@types/bootstrap": "~5.0.17",
+    "@types/jest": "~27.4.0",
+    "@types/node": "~17.0.18",
```

Comment:
musst eventuell dem renovate bot in der config sagen, dass ~ erlaubt ist. Bin aber kein Fan davon